core/src/avm2/globals/flash/net.rs

```diff
@@ -39,10 +64,20 @@ pub fn navigate_to_url<'gc>(
         )?)),
         url => {
             let url = url.coerce_to_string(activation)?;
+            let method = request
+                .get_public_property("method", activation)?
+                .coerce_to_string(activation)?;
+            let method =
+                NavigationMethod::from_method_str(&method).unwrap_or(NavigationMethod::Get);
```

Comment:
After rebasing on #15145 this can just be `unwrap` or `expect`.

Reply:
Good call! Fixed in https://github.com/ruffle-rs/ruffle/pull/15129/commits/7fa96c36656b4d68bb9b721400d88f42fe6190b6